app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Please create two methods for the post model which are given in the following [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=A%20method%20that%20updates,for%20a%20given%20post.).